docs/data/joy/customization/theme-colors/theme-colors.md

```diff
@@ -0,0 +1,284 @@
+# Theme colors
+
+<p class="description">Learn about the theme's default colors and how to customize them.</p>
+
+## Default tokens
+
+The table below lists all the default tokens and their values in light and dark color schemes.
+Some tokens reuse values from other tokens using the [`var(--*)`](https://developer.mozilla.org/en-US/docs/Web/CSS/var) syntax.
+
+{{"demo": "PaletteThemeViewer.js", "bg": "inline"}}
```

Comment:
I would recommend using the colors as a background on the cells, and allow copying the token when clicking/hovering over the cell. Once upon a time I was doing something like this for Fluent UI, I managed to find a link to it :D: https://fluentsite.z22.web.core.windows.net/0.65.0/color-schemes

Reply:
> I would recommend using the colors as a background on the cells

Honestly, I don't think this would be that much different from the current look and I guess it needs JS to calculate the correct contrast (between white and black text color).

> and allow copying the token when clicking/hovering over the cell

Yep, this would be useful.